src/types/markers.js

```diff
@@ -565,7 +565,7 @@ export type ChromeInstantTraceEventPayload = {|
 // ph: 'B' | 'E' in the Trace Event Format
 export type ChromeDurationTraceEventPayload = {|
   type: 'tracing',
-  category: 'FromChrome',
+  category: string,
```

Comment:
For some reason this started to error after I added `FallbackEndEvent` 🤔 Indeed there's a place where we pass an arbitrary string...

Reply:
Oh interesting. Good to put it as string then if we have a place where we pass an arbitrary string.